hypervisor/include/arch/x86/cpu.h

```diff
@@ -310,7 +310,7 @@ void start_cpus();
 /* This macro writes the stack pointer. */
 #define CPU_SP_WRITE(stack_ptr)                             \
 {                                                           \
-	uint64_t rsp = (uint64_t)stack_ptr & ~(CPU_STACK_ALIGN - 1);  \
+	uint64_t rsp = (uint64_t)stack_ptr & ~(CPU_STACK_ALIGN - 1U);  \
```

Comment:
This should be 1UL.. Not?

Reply:
My bad. CPU_STACK_ALIGN and the '1' here should all be uint64_t.